Frontend/src/components/mentor/mentor-dashboard/MentorDashboard.jsx

```diff
@@ -1,10 +1,10 @@
 import { useState} from 'react'
-import { UserContext } from '../UserContext.jsx';
+import { UserContext } from '../../../UserContext.jsx';
 import { Link } from 'react-router-dom';
 import './MentorDashboard.css';
-import Calendar from './Calendar.jsx';
-import Matches from './Matches.jsx';
-import Requests from './Requests.jsx';
+import Calendar from '../../Calendar.jsx';
+import Matches from '../mentor-matches/MentorMatches.jsx';
+import Requests from '../mentor-requests/Requests.jsx';
 
 function MentorDashboard() {
```

Comment:
I feel you have several separate similar components specific for mentor and mentee. I think you can share some components for both Mentor and Mentee and distinguish by their roles. Maybe consider as a future better engineering work.

Reply:
Great idea. I tried to reuse as many components as possible for both, but for some I created new ones that are kind of similar because the mentor and mentee have different views. Some things the mentor has the mentee won't.